data/dwarfs-compress-handler.desktop

```
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=DwarFS Autocompress
+TryExec=mkdwarfs
+Exec=mkdwarfs --input %f
```

Wouldn't it be possible to just change this to
```suggestion
Exec=mkdwarfs --input %f --output %f.dwarfs
```
and completely avoid the `mkdwarfs` change?